### ISSUE TITLE
Node selection flips 'randomly' after node close

### ISSUE THREAD

**Issue by mxmilkiib** [baskerville/bspwm#1189]
If I have two nodes open, then I repeatedly open and close a third, the direction of the node selection after the closure of the third node appears to alter randomly.

[Here's a video](https://www.youtube.com/watch?v=cVPPYvWOVZE). I've tried commenting out various config entries (link in the video description) and looking through the CHANGELOG, but I haven't been able to figure this out.

**Comment by emanuele6** [comment]
Steps for reproducing this bug?
Can you reproduce this with `bspwm`'s default configuration? (I can't!)
Do you use a compositor? If you do, what's your configuration for the compositor you use?
Can you reproduce this bug only with terminal windows or with other windows too?
What terminal emulator is that?

> https://github.com/mxmilkiib/dotfiles/blob/cdc9d7af47653fe682d918df50f002add30259a7/bspwm/.config/bspwm/bspwmrc

In the `bspwmrc` file you linked in the description of your video, you are configuring some settings that have been remove for a lot of time: e.g. `auto_cancel` and `wm_name`.

**Comment by mxmilkiib** [comment]
The problem is apparently terminal related.

Here is the default config doing it with rxvt-unicode: https://www.youtube.com/watch?v=5YX3Xu1q0SE

I changed $TERMINAL from urxvt to xterm and restarted, now the problem is gone. I'll try with termite next restart.

No compositor, and I've used bspwm for a while and there isn't a config validation system I know of yet though I've cleaned it up now. AUR bspwm-git,`0.9.10.r5.g760b148-1`.
